srcpkgs/virtuoso/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'virtuoso'
 pkgname=virtuoso
-version=7.2.5.1
-revision=3
+version=7.2.10
+revision=1
 archs="x86_64* aarch64* ppc64le*"
 build_style=gnu-configure
-configure_args="--enable-static"
-hostmakedepends="automake libtool flex gperf net-tools"
+configure_args="--without-internal-zlib --with-readline --with-layout=debian"
+hostmakedepends="automake libtool flex gperf net-tools which"
 makedepends="openssl-devel libldap-devel readline-devel libxml2-devel mit-krb5-devel"
+checkdepends="procps-ng unzip curl wget"
 short_desc="Scalable cross-platform server of virtuoso"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-2"
+license="GPL-2.0-only"
 homepage="http://virtuoso.openlinksw.com/wiki/main/Main/"
```

Comment:
Redirects to https://vos.openlinksw.com/owiki/wiki/VOS/

Reply:
Updated to the new one. There's also https://virtuoso.openlinksw.com/ but that's not for the open source version.